lib/plugins/TaskRunner/LAMPApp.js

```diff
@@ -70,7 +70,7 @@ class LAMPApp extends Script {
 
   addScriptHeader() {
     this.script = this.script.concat([
-      'sleep 3',
+      'READY=0; while ! `nc -z 127.0.0.1 3306` && [ $READY -lt 60 ]; do echo "Waiting for MySQL..."; READY=$((READY + 1)); sleep 1; done; if `nc -z 127.0.0.1 3306`; then echo "MySQL is ready."; else echo "MySQL failed to start!"; exit 1; fi;',
```

Comment:
If we're going to put a timeout for MySQL, it should be a config setting in our YAML. Since this is open source, it might be nice to set the mysql port as a config setting too.